### ISSUE TITLE
Accept lower-case SAN

### ISSUE THREAD

**Issue by rpdelaney** [trevorbayless/cli-chess#16]
### What feature would you like to see in cli-chess?

When typing moves it would be nice to be able to do `nb3` , but this is rejected and `Nb3` is required.

**Comment by trevorbayless** [comment]
Correct, this is the intended functionality for SAN input as it follows proper chess notation. Considering this program allows input of SAN, UCI, and LAN this could get overly complicated to implement. Currently all move input is sent over to python-chess for parsing and validation and with this change cli-chess would need to handle the upfront parts of that as well.

Let me ponder on this one for a little bit to see if there is a straightforward way to implement it. I might be able to convert all legal moves and move input to lowercase and look for a match, but would need to take some performance measurements on this approach.
